patterns/minimising-data-size-transmitted-with-reduction.md

```diff
@@ -0,0 +1,37 @@
+# Minimising the size of data transmitted with reduction
+
+## Version
+1.0
+
+## Submitted By
+Sarah Hsu (@greenhsu123)
+
+## Published Date
+TBD
+
+## Intent
+Minimising the size of data transmitted with reduction 
+
+## Tags
+Cloud, Networking, Cloud Engineer, Small
+
+## Problem
+From an energy-efficiency perspective, it's better to minimise the size of the data transmitted so that less energy is required because the network traffic is reduced. 
+
+## Solution
+Minimise the size of data transmitted by only sending properties or values deemed necessary. 
+
+## SCI Impact
+`SCI = (E * I) + M per R`
+
+Concerning the SCI equation. Reducing the distance will impact one part:
+
+- `E`: We reduce the total electricity required by reducing network traffic. 
```

Comment:
What do you think about adding `M` here as well for the storage requirements. It doesn't have to do with Network specifically but transmitted data like telemetry is often stored and retained somewhere. Reducing payload size would reduce the embodied carbon responsible for storing it. This may also work better in its own pattern as well, curious what you think.

Reply:
If we were to consider adding `M`, I think it would be better to split it into a different pattern because both the solution and problem would address the intent from a storage perspective... 
What do you think?

Reply:
That works, resolving this and we can create a new pattern for the storage component that is separate.